tests/test_cohort_joiner.py

```diff
@@ -49,14 +73,25 @@ def test_write_csv(self, tmp_path, dataframe):
         # expect their input to resemble cohort-extractor's output. For this reason, we
         # read the CSV that we wrote, and test that the header row (the zeroth row) is
         # correct.
-        with csv_path.open(newline="") as f:
+        with open_func(csv_path, "rt", newline="") as f:
             lines = list(csv.reader(f))
         assert list(dataframe.columns) == lines[0]
 
-    def test_write_xlsx(self, tmp_path, dataframe):
+    def test_write_feather(self, tmp_path, dataframe):
+        feather_path = tmp_path / "input.feather"
+        cohort_joiner.write_dataframe(dataframe, feather_path)
+        feather_path.exists()
```

Comment:
Are you missing an `assert` here, or does `.exists()` raise an error?

Reply:
Good spot! 👀 I'll add `assert` here and elsewhere, where I call `.exists()`; rebase; and force push.

Reply:
It can be a good idea to make sure that you've seen every test fail, so that you can be more confident that they're passing for the reasons you think!